### PR TITLE
add applied filters overview

### PR DESCRIPTION
**Pull request by hiagolcm**
#### What problem is this solving?

Add an overview of the applied filters.

#### How to test it?

[Workspace](https://filteroverview--storecomponents.myvtex.com/apparel---accessories/gold/grey/pink/silver/white?fuzzy=0&map=department,color,color,color,color,color&operator=and)

#### Screenshots or example usage:

image

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -231,7 +231,8 @@ Structure the `search-result-layout` or the `search-result-layout.customQuery`,
...
 | `trackingId` | `string` | ID to be used in Google Analytics to track store metrics based on the Search Result block. |  `Search result` | 
 | `mobileLayout`      | `Object` | Controls how the search results page will be displayed to users when using the mobile layout. Possible props and their respective values can be found below.                                                                                                                | `undefined`              |
-| `thresholdForFacetSearch` | `number` | Minimum number of facets that must be displayed on the interface for a search bar to be displayed. If you declare `0`, the search bar will always be displayed. |  `undefined` | 
+| `thresholdForFacetSearch` | `number` | Minimum number of facets that must be displayed on the interface for a search bar to be displayed. If you declare `0`, the search bar will always be displayed. |  `undefined` |
+| `showAppliedFiltersOverview` | `boolean` | Whether the list of applied filters should appear below the facet name (`true`) or not (`false`). | `true` |
```
Comment (klzns) [inline docs/README.md:235]: Boolean prop 👀

File: react/components/AccordionFilterItem.js [hunk 2]
```diff
@@ -80,6 +85,16 @@ const AccordionFilterItem = ({
               <span className={`${handles.accordionFilterItemIcon} fr`}>
                 <IconCaret orientation="down" size={10} />
               </span>
+              {showAppliedFiltersOverview && quantitySelected > 0 && (
+                <div
+                  className={classNames(
+                    handles.accordionSelectedFilters,
+                    'f6 c-action-primary'
```
Comment (klzns) [inline react/components/AccordionFilterItem.js:92]: If we can't click on the element, it doesn't make sense to use `c-action-primary`

See: https://styleguide.vtex.com/#/Styles?id=section-colors

File: react/components/FilterOptionTemplate.js [hunk 3]
```diff
@@ -231,6 +233,19 @@ const FilterOptionTemplate = ({
             )}
           </div>
         </div>
+        {showAppliedFiltersOverview && filters && !selected && (
+          <div
+            className={classNames(
+              handles.filterSelectedFilters,
+              'f6 c-action-primary'
```
Comment (klzns) [inline react/components/FilterOptionTemplate.js:240]: Same

**Review by klzns — CHANGES_REQUESTED** [review]
Missing changelog

**Review by guerreirobeatriz — APPROVED** [review]

File: docs/README.md (hunk 1 above)
Comment (hiagolcm) [reply docs/README.md]: Done! I also moved the prop to the FilterNavigator component

**Review by icazevedo — APPROVED** [review]

**Review by klzns — APPROVED** [review]

**Pull request merged**